.github/workflows/gemmini.yml

```diff
@@ -25,6 +25,7 @@ jobs:
           apt-get remove -y cmake
           apt-get install -y ninja-build
           python -m pip install -r requirements.txt
+          python -m pip install -r dev-requirements.txt
```

Comment:
I recommend splitting the requirements into two files: `exo-dev-requirements.txt` (for people who wants to develop exo) and `exo-requirements` for people who just want to use exo. Then, you can include them both from a `requirements.txt` file using the `-r <file>` format: https://pip.pypa.io/en/stable/reference/requirements-file-format/#referring-to-other-requirements-files

This way any one person (dev or just user) need to run one command to install everything and also dependabot can discover all the requirements (it looks only for `requirements.txt`).

Reply:
I think it makes sense to have requirements.txt just for users, instead of including the dev dependencies. CI will look at tox.ini for test requirements separately, so I don't think that's an issue?

Reply:
Yeah but dependabot would then not see the dev requirements? https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#pip-and-pip-compile

Reply:
From this discussion ( https://github.com/dependabot/dependabot-core/issues/3940#issuecomment-1227005267 ) and dependabot's source code ( https://github.com/dependabot/dependabot-core/blob/main/python/helpers/lib/parser.py#L84 ) it seems like they're scanning all *.txt files and is not relying on "requirements.txt" pattern match, so I think having two separate txt files is fine

Reply:
Ah I see. That's not clear from the documentation. Good to know!